FEM/pcs_dm.cpp

```diff
@@ -795,13 +795,16 @@ double CRFProcessDeformation::Execute(int loop_process_number)
 				{
 					pcs_absolute_error[0] = NormU;
 					pcs_relative_error[0] = pcs_absolute_error[0] / Tolerance_global_Newton;
-					cpl_max_relative_error = pcs_relative_error[0];
+					// To be improved by overloading CRFProcess::CalcIterationNODError
+					cpl_max_relative_error = pcs_absolute_error[0]; //pcs_relative_error[0];
```

Comment:
two comments
- why cannot you use `pcs_relative_error[0]`?
- it's quite confusing if `cpl_max_relative_error` is actually absolute error. isn't there `cpl_max_abs_error`?

Reply:
`cpl_max_relative_error` is only set here under `if (ite_steps == 1)`. If `cpl_max_relative_error` is set to `pcs_relative_error[0]`( = ` (pcs_absolute_error[0] / Tolerance_global_Newton`), the coupling convergence criterion of  `cpl_max_relative_error < 1`  is never satisfied  for some cases. Anyway, this implementation is not correct. Therefore, I made this change just to make sure that computation of the modelling of  M related coupled processes can  keep running,  while its coupling convergence is controlled by other processes. As you can see in the comment above the changed line, the method in   CRFProcess::CalcIterationNODError will be used to calculate the coupling error of M process.

Reply:
@norihiro-w  Thanks for your comments. I am going to merge this PR.